sql/changes/1.6/menu_item.sql

```diff
@@ -0,0 +1,11 @@
+DROP TYPE IF EXISTS menu_item CASCADE;
+CREATE TYPE menu_item AS (

```

Comment:
I'm not sure I understand this change: we were returning a flat list where each collection element refers to its parent by `id`. What - on the server side - is going to populate the `children`? And if we return an array of children, then from the server's perspective, things are no longer flat, because you need recursive/additional queries to determine which are the children...